app/updater.js

```diff
@@ -85,6 +86,35 @@ var scheduleUpdates = () => {
     }, appConfig.updates.appUpdateCheckFrequency)
   }
 
+  // Linux doesn't have an auto-update mechanism.
+  // Instead, deliver a periodic nag linking to instructions
+  if (isLinux() && appConfig.updates.deprecatedNagFrequency) {
+    electron.ipcMain.on(messages.NOTIFICATION_RESPONSE, (e, message, buttonIndex, persist) => {
+      appActions.hideNotification(message)
+    })
+
+    const showDeprecationNotice = () => {
+      appActions.showNotification({
+        buttons: [
+          {text: 'OK'}
+        ],
+        options: {
+          persist: false,
+          advancedText: 'See instructions to upgrade to the latest version',
+          advancedLink: 'https://brave-browser.readthedocs.io/en/latest/installing-brave.html#linux'
```

Comment:
This is blocked on the doc text being updated with the new signing keys: https://github.com/brave/brave-browser/issues/1826